docs/recipes/authentication.md

```diff
@@ -35,6 +35,7 @@ service.
     username: null
   }
 })
+@Injectable() // Angular9 IVY Update, check detail in here; https://github.com/ngxs/store/blob/master/docs/advanced/ivy-migration-guide.md
```

Comment:
I think the comments here are unnecessary

Reply:
Let me change, and I will make same update for all other recipes.